### DISCUSSION TITLE
Other themes

### DISCUSSION THREAD

**Discussion by Autoparallel** [tefkah/zotero-night#62] [Ideas]
Is there any move towards allowing other themes outside of Nord? For instance, Dracula, Gruvbox, Solarized, etc.? Some CSS available here: https://github.com/lassekongo83/adw-colors

This would be a great feature!

**Comment by tefkah** [comment]
This is planned, but I'm really bad at working with Zotero's settings system and don't have a lot of time, so it will come just take a while!
